base/src/com/google/idea/blaze/base/command/info/BlazeInfo.java

```diff
@@ -114,7 +114,7 @@ private static String getOrThrow(ImmutableMap<String, String> map, String key) {
 
   abstract ImmutableMap<String, String> getBlazeInfoMap();
 
-  public String get(String key) {
+  protected String get(String key) {
```

Comment:
We were using this method in our extension to load the java-home from bazel info. Was there a reson to restrict its visibility?

Reply:
Yes. I made accessors for individual values and restricted the set of values pulled from bazel info when i merged the two calls to it generated by the fact that i needed to pull info about startlark state (to see if bzlmod and other similar flags are set or not).

I also switched the callers of that to use the accessors. Not sure how i missed yours. What is the context of that miss ?

Reply:
This PR also moved to a while list of things pulled from `bazel info`. I'm pretty sure java-home was not there since there were no uses of it in the plugin codebase. I can add that and make an accessor for it. Would that help you ?

Reply:
Its a internal plugin we have on top of bazel, so no easy to see it was being used :)

I'll let you know, Right not at least its only a compile error and should not be affecting runtime. I'm seeing whether we can use the projects SDK for this usecase instead of using this from bazelInfo.

Reply:
Went ahead and created a PR to add java-home to the info cache: https://github.com/bazelbuild/intellij/pull/6810 
Our use case does require to use the same java that bazel uses. Thanks for the explanation, I was able to understand the changes better.